.github/workflows/app_build.yml

```diff
@@ -0,0 +1,304 @@
+name: C/C++ CI for OpenStudioApplication
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  OS_SDK_VERSION: 3.0.1
```

Comment:
`OS_SDK_VERSION` should be pulled from a file in source, something like `cat OSVersion.txt`, this is a bit too buried

Reply:
Or pull it from `CMakeLists.txt` like you did for app version.

Reply:
I pulled it from FindOpenStudioSDK.cmake...
At which point you start to wonder whether getting CMake to download the tar.gz instead of using the bash script I wrote would make more sense...